src/olympia/amo/tests/test_views.py

```diff
@@ -377,6 +378,12 @@ def test_cors_api_v4(self):
         assert not response.has_header('Access-Control-Allow-Credentials')
         assert response['Access-Control-Allow-Origin'] == '*'
 
+    def test_cors_excludes_accounts_session_endpoint(self):
+        assert re.match(
+            settings.CORS_URLS_REGEX,
+            reverse('v4:accounts.session'),
```

Comment:
This should probably use `reverse_ns` to make it work for all api versions maybe. Nitpick though-

Reply:
I tried but `reverse_ns` returns an absolute URL and the regex is for a (relative) path.

Reply:
yeah... we're quite careful to avoid using django's reverse and `v4:xxx` syntax for API urls.  
`from rest_framework.reverse import reverse` should have been used instead though - it doesn't return absolute urls by default.

Reply:
Let me fix that in a follow-up patch, thanks!